api_compatibility_tests/tox.ini

```diff
@@ -23,6 +23,7 @@ deps =
     awses_cli_1.7.0: -rcompatibility-requirements/1.7.0
     awses_cli_1.8.0: -rcompatibility-requirements/1.8.0
     awses_cli_1.9.0: -rcompatibility-requirements/1.9.0
+    awses_cli_1.9.0: -rcompatibility-requirements/1.9.1
```

Comment:
There is a typo here. It should be `awses_cli_1.9.1: -rcompatibility-requirements/1.9.1`.

Reply:
Good catch, I'll get a fix out.